plugins/regex_revalidate/regex_revalidate.c

```diff
@@ -77,9 +79,10 @@ typedef struct invalidate_t {
 
 typedef struct {
   invalidate_t *invalidate_list;
-  char *config_file;
+  char *config_path;
```

Comment:
Weren't we going to convert this plugin to C++?  Would be easier to maintain if this was a std::string.

Reply:
perhaps in another pr.